tests/tests_integration/test_api/test_simulators/utils.py

```diff
@@ -0,0 +1,9 @@
+from cognite.client._cognite_client import CogniteClient, Response
+
+
+def update_logs(cognite_client: CogniteClient, log_id: int, payload: list[dict]) -> Response:
+    items = {"items": [{"id": log_id, "update": {"data": {"add": payload}}}]}
+    return cognite_client.post(
+        url=f"/api/v1/projects/{cognite_client.config.project}/simulators/logs/update",
```

Comment:
this url (with project) should be construct-able from `cognite_client.simulators.logs`, no?